### PR TITLE
fix(get_group0_members): catch exception for cql commands

### PR DESCRIPTION
**Pull request by aleksbykov**
run_cqlsh command could failed during getting group0 members and raised exception failed the nemesis thread during cluster health validation.

Process exception raised in get_group0_members and publish appropriate event

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -3042,56 +3042,63 @@ def get_token_ring_members(self) -> list[dict[str, str]]:
...
+            for line in result[3:]:
+                member = line.split("|")
+                if not member or len(member) != 2:
+                    break
+                group0_members.append({"host_id": member[0].strip(),
+                                       "voter": member[1].strip() == "True"})
+        except Exception as exc:  # pylint: disable=broad-except
+            err_msg = f"Get group0 members failed with error: {exc}"
+            self.log.error(err_msg)
+            InfoEvent(message=err_msg, severity=Severity.ERROR).publish()
```
Comment (fruch) [inline sdcm/cluster.py:3100]: healthchecks has it's own set of events per the specific failure.
as a followup you might want to align with the rest of them

**Review by fruch — APPROVED** [review]
LGTM

File: sdcm/cluster.py [hunk 2]
```diff
@@ -3042,56 +3042,63 @@ def get_token_ring_members(self) -> list[dict[str, str]]:
...
+            # Start parsing from 4th line
+
+            for line in result[3:]:
+                member = line.split("|")
+                if not member or len(member) != 2:
+                    break
+                group0_members.append({"host_id": member[0].strip(),
+                                       "voter": member[1].strip() == "True"})
+        except Exception as exc:  # pylint: disable=broad-except
+            err_msg = f"Get group0 members failed with error: {exc}"
```
Comment (fgelcer) [inline sdcm/cluster.py:3098]: for logs, better using lazy logging

**Review by fgelcer — APPROVED** [review]
other than nitpit, LGTM

**Review by soyacz — APPROVED** [review]

**Pull request merged**